### PR TITLE
Add deposit events

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

File: packages/arb-bridge-eth/contracts/bridge/Inbox.sol [hunk 1]
```diff
@@ -34,6 +34,12 @@ contract Inbox is IInbox {
 
     IBridge public override bridge;
 
+    event DepositEthRetryable(
```
Comment (hkalodner) [inline packages/arb-bridge-eth/contracts/bridge/Inbox.sol:37]: Let's skip the Inbox.sol changes for now in the interest of finalizing for the testnet

File: packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/EthERC20Bridge.sol [hunk 2]
```diff
@@ -43,6 +43,30 @@ contract EthERC20Bridge is L1Buddy {
     address private immutable l2TemplateERC20;
 
 
+    event DepositERC20(
+        address indexed destination,
```
Comment (hkalodner) [inline packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/EthERC20Bridge.sol:47]: Maybe also include sender?

File: packages/arb-ts/src/lib/abi/ArbRetryableTx.d.ts [hunk 3]
```diff
@@ -27,7 +27,6 @@ interface ArbRetryableTxInterface extends ethers.utils.Interface {
     'getBeneficiary(bytes32)': FunctionFragment
     'getKeepalivePrice(bytes32)': FunctionFragment
     'getLifetime()': FunctionFragment
-    'getSubmissionPrice(uint256)': FunctionFragment
```
Comment (hkalodner) [inline packages/arb-ts/src/lib/abi/ArbRetryableTx.d.ts:30]: This shouldn't have been deleted. You might be on an old ArbOS version

**Review by hkalodner — CHANGES_REQUESTED** [review]

**Comment by DZGoldman** [comment]
@hkalodner added the arb-ts updates to deposits to this branch  (https://github.com/OffchainLabs/arbitrum/pull/924/commits/4c7ae662e1020bb9e669e0705103e630a6ef4bd9)

**Review by hkalodner — APPROVED** [review]
LGTM

**Pull request merged**